lib/plugins/aws/lib/updateStack.js

```diff
@@ -94,6 +94,24 @@ module.exports = {
       });
   },
 
+  validateStackTemplate() {
+    const compiledTemplate = _.cloneDeep(
+      this.serverless.service.provider.compiledCloudFormationTemplate);
+
+    this.serverless.cli.log('Validating Stack...');
+    const params = {
+      TemplateBody: JSON.stringify(compiledTemplate),
+    };
+
+    return this.provider.request(
+      'CloudFormation',
+      'validateTemplate',
+      params,
+      this.options.stage,
+      this.options.region
+    ).then((cfData) => this.monitorStack('validate', cfData));
```

Comment:
Check if `monitorStack` supports the `validate` action. If not, it should be added there.